test/src/hls-adapter.spec.js

```diff
@@ -483,6 +484,81 @@ describe('HlsAdapter Instance - change media', function() {
       });
     });
   });
+
+  it('should fire FRAG_LOADED', done => {
+    hlsAdapterInstance = HlsAdapter.createAdapter(video, source1, config);
+    hlsAdapterInstance.addEventListener(EventType.FRAG_LOADED, event => {
```

Comment:
Wrap this block by `try catch`. see https://github.com/kaltura/playkit-js-hls/blob/f85128d2e3604b1755ae19d15b986bc1acaf88e0/test/src/hls-adapter.spec.js#L317

Reply:
done